packages/components-kit/src/components/ComponentForm/ComponentForm.component.js

```diff
@@ -67,7 +68,8 @@ export function keepOnlyDatasetMetadataProperties({ body, properties }) {
 export class TCompForm extends React.Component {
 	constructor(props) {
 		super(props);
-		this.state = {};
+		const properties = props.state.get('properties');
+		this.state = { properties: properties && properties.toJS() };
```

Comment:
I don t see how state properties could exists at this stage.
So let s remove it